package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymer-css-build",
-  "version": "0.0.6",
+  "version": "0.0.7",
```

Comment:
Don't update the version number in a pull request, we'll do that when we stamp a new release based on any other unreleased changes.